Corgibytes.Freshli.Cli/Program.cs

```diff
@@ -91,6 +97,19 @@ public static CommandLineBuilder CreateCommandLineBuilder()
                 await next(context);
             })
             .AddMiddleware(async (context, next) => { await LogExecution(context, next); })
+            .AddMiddleware(async (context, next) =>
+            {
+                var workerCount = context.ParseResult.GetOptionValueByName<int>("workers");
+                HangfireBackgroundService?.StopAsync(CancellationToken.None);
```

Comment:
Instead of stopping the service here, should we instead just not start it in the `FreshliServiceBuilder`?

If we decide to keep this stop call here, we should make this call with `await` so that we don't risk calling `StartAsync` before `StopAsync` has finished.

Reply:
I'll give the not starting it a try. Adding the `await` shows a memory warning:
![image](https://user-images.githubusercontent.com/7047894/194550668-27bbe021-29da-473f-b990-18c18eb3cafc.png)

Reply:
Oh without await as well.